README.md

```diff
@@ -31,8 +31,8 @@ let package = Package(
 )
 ```
 
-Then you can start using Noora. The interface is simple and intuitive.
-You need to create an instance of `Noora`, and every component available is represented as a function in the `Noora` instance.
+Then you can start using Noora, the interface is simple and intuitive.
```

Comment:
```suggestion
Then you can start using Noora.
```

Let's let the reader decide what they think of the API – this part of the sentence doesn't bear any value, everybody thinks their interface is simple and intuitive 😄